src/usr/local/www/interfaces.php

```diff
@@ -274,7 +274,10 @@ function remove_bad_chars($string) {
 		$pconfig['type6'] = "slaac";
 		break;
 	case "dhcp6":
-		$pconfig['dhcp6-duid'] = $wancfg['dhcp6-duid'];
+		if(empty($config['system']['global-v6duid'])) {
+			unset($wancfg['dhcp6-duid']);
+		}
```

Comment:
Please explain what you mean by 'global option', What I am trying to prevent is the user setting 'dhcp6-duid' when there  is no valid 'global-v6duid'. I would have preferred the option of masking the select box, if that is possible, thus making it impossible to select the option in the first place.